### ISSUE TITLE
it removed automatically things i already fixed and how can i download the code i generated

### ISSUE THREAD

**Issue by irfansaeedkhan** [stackblitz/bolt.new#320]
### Describe the bug

it removed automatically things i already fixed  when i open it again and how can i download the code i generated , there is no zip or push to github option

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-5nsulr

### Steps to reproduce

write promt to generate a code , then ask for few upgrades and fixes and it will again show same error if open again the same chat

### Expected behavior

what i fixed till now it should keep it as it is, i dont have to fix bug again , also i want my code in my ocal system i cannot copy paste file by file

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by Alohacop** [comment]
You can open in StackBlitz and then you can download as a ZIP or push to GitHub. See attached screenshot. Good luck brother!

![Step 1](https://github.com/user-attachments/assets/3422e6c8-649e-444f-80f0-fceff9f5307f)
![Step 2](https://github.com/user-attachments/assets/ffd5a84e-1155-4d51-816f-9dacb1da98c9)

**Comment by Faaiqster** [comment]
> How do you push it to an existing repository. I'm trying to use auto deploy in netlify from GitHub. I push from Bolt to StackBlitz. I want to auto deploy. I have a domain connected but when I deploy from Bolt it does not use the main domain account.

Wondering the same as well.

**Comment by plugmakr** [comment]
Yeah, this needs to be addressed. You can change the Git repository in the netlify settings but it is a pain.
